### PR TITLE
Add snippet-test-labeling enforcement script

### PR DESCRIPTION
**Pull request by ace-n**
@fhinkel FYI - this should be an _optional_ repo check for now.

(I'm planning to make it required on various directories as I bring them up to `passing` status.)

cc @jmdobry as FYI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 **/node_modules/**
 npm-debug.log
 coverage/
+enforcer.sh
```
Comment (fhinkel) [inline .gitignore:4]: This shouldn't be here.

**Comment by fhinkel** [comment]
I'm not a fan of adding failing tests. Do you have a doc with a time line and ownership for when tests will be fixed?

File: .gitignore (hunk 1 above)
Comment (ace-n) [reply .gitignore:4]: This script is downloaded with each run (to keep it up to date) - so it shouldn't be checked in to the repo.

Comment (fhinkel) [reply .gitignore:4]: But it's downloaded to CI, not locally.

**Comment by fhinkel** [comment]
LGTM as discussed off line.

File: .gitignore (hunk 1 above)
Comment (ace-n) [reply .gitignore]: I don't feel strongly about this, so I'm fine reverting.

**Comment by ace-n** [comment]
@fhinkel FYI, this test will hopefully a) show up and b) start passing now.

Please **do not** merge this PR if it doesn't. Thanks!

**Comment by fhinkel** [comment]
Test shows up, but it's failing.

**Comment by ace-n** [comment]
@fhinkel thanks - I merged [this PR](https://github.com/GoogleCloudPlatform/repo-automation-playground/pull/22) just now, which should fix things. :)

**Pull request merged**